python/lsst/ts/imsim/opdMetrology.py

```diff
@@ -89,6 +78,8 @@ def setDefaultLsstWfsGQ(self):
         self.wt = np.array([1.0, 1.0, 1.0, 1.0])
         wfsFieldX, wfsFieldY, sensorIds = self.getDefaultLsstWfsGQ()
         self.fieldX, self.fieldY = (wfsFieldX, wfsFieldY)
+        # Convert from CCS to ZCS for current OFC
+        self.fieldX = -1.0 * np.array(self.fieldX)
         self.sensorIds = sensorIds
 
     def getDefaultLsstWfsGQ(self):
```

Comment:
One general comment, I know it may take some time, but it might be nice to add typing to this package? If so, we could start in this PR with the functions here? Alternatively we can also do it in another PR?

Reply:
Added it to the list of things in this issue: https://jira.lsstcorp.org/browse/DM-40355?filter=21514

Reply:
If you have more formatting comments let's add them there.